test/functional/test_framework/test_framework.py

```diff
@@ -814,7 +814,7 @@ def _initialize_chain(self):
                     cache_node_dir,
                     chain=self.chain,
                     extra_conf=["bind=127.0.0.1"],
-                    extra_args=['-disablewallet'],
```

Comment:
@BrandonOdiwuor What does the `-disablewallet` mean?

Reply:
you can follow the conversation under the issue: https://github.com/bitcoin/bitcoin/issues/27593